### PR TITLE
Fllowing tour

### PR DESCRIPTION
**Pull request by Rittamx031**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/72787)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by Rittamx031** [comment]
ready

**Comment by ghost** [comment]
Add link ticket redmine vào nhé

File: app/controllers/relationships_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  def destroy
+    current_user.unfollow_tour(@tour)
+    flash.now[:success] = t("tours.unfollow_success")
+  end
+
+  private
+
+  def load_tour
+    @tour = Tour.find_by id: params[:id]
```
Comment (ghost) [inline app/controllers/relationships_controller.rb:18]: Thay vì check tồn tại tour ở dưới model thì check ngay tại đây luôn đi em

File: app/views/admin/layouts/_side_navbar.html.erb [hunk 2]
```diff
@@ -25,8 +25,8 @@
         <div class="dropdown-menu">
           <%= link_to t("dashboard.link.categorys"), admin_tour_categories_path, class: "dropdown-item" %>
           <%= link_to t("dashboard.link.new_categorys"), new_admin_tour_category_path, class: "dropdown-item" %>
-        </div>
-      </li>
-    </ul>
-  </div>
-</nav>
+          </div>
```
Comment (ghost) [inline app/views/admin/layouts/_side_navbar.html.erb]: Là sao?

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -65,27 +65,27 @@
...
+                  <%= link_to t("user.headers.account_link.register"), signup_path, class: "dropdown-item" %>
+                </div>
+              <% end %>
+            </div>
+            <div class="nav-item dropdown">
+              <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Language</a>
               <div class="dropdown-menu border-0 rounded-0 m-0">
-                <%= link_to t("user.headers.account_link.login"), login_path, class: "dropdown-item" %>
-                <%= link_to t("user.headers.account_link.register"), signup_path, class: "dropdown-item" %>
+                <%= link_to "English", url_for(locale: "en"), class: "dropdown-item" %>
```
Comment (ghost) [inline app/views/layouts/_header.html.erb]: I18n cho tên

**Comment by Rittamx031** [comment]
ready

**Comment by ghost** [comment]
merged!

**Pull request merged**